bitnami/rabbitmq/Chart.yaml

```
@@ -23,4 +23,4 @@ name: rabbitmq
 sources:
   - https://github.com/bitnami/bitnami-docker-rabbitmq
   - https://www.rabbitmq.com
-version: 8.20.1
+version: 8.21.0
```

Please bump the patch version instead:
```suggestion
version: 8.20.2
```

---

Sure. Fixed and pushed, thanks.